AWSClientRuntime/Tests/Sigv4/SigV4SigningTests.swift

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import AwsCommonRuntimeKit
+import ClientRuntime
+import XCTest
+
+@testable import AWSClientRuntime
+
+extension Formatter {
+    // Inspired from:
+    // https://stackoverflow.com/questions/28016578/how-can-i-parse-create-a-date-time-stamp-formatted-with-fractional-seconds-utc
+    static let iso8601withFractionalSeconds: DateFormatter = {
```

Comment:
I believe we have one of these already in the ClientRuntime?

Reply:
Actually looks like this isn't being used? Remove?

Reply:
It's being used in the unit test for that specific date format, however, the only reason this exists is because i was copying a unit test from the iOS sdk repo.  I agree though, let me update the test so that we don't need this.

Reply:
k I believe we do have this exact formatter in ClientRuntime if you do need it.

Reply:
It was slightly different, but i updated the test to avoid having to use it.